### DISCUSSION TITLE
Can't Launch Program

### DISCUSSION THREAD

**Discussion by DaGwhite** [nutdotnet/WinNUT-Client#138] [Q&A] [answered]
I installed this last night, and was trying to figure out how to get it to connect to my NetGear ReadyNAS 312, which is connected to a CyberPower UPS via USB. I gave up & went to bed. I don't know if I need to log into the NAS, so I thought I would remove that info.

When I tried to open the program this morning to try something else, it immediately threw an error. Here is the crash report:
-------------------
WinNUT Bug Report
Os Version : 6.2.9200.0
WinNUT Version : 2.1.7740.35837

WinNUT Parameters :
{
"Delay": 0,
"AutoReconnect": "True",
"MinInputVoltage": 210,
"MaxInputVoltage": 270,
"FrequencySupply": 0,
"MinInputFrequency": 40,
"MaxInputFrequency": 60,
"MinOutputVoltage": 210,
"MaxOutputVoltage": 250,
"MinUPSLoad": 0,
"MaxUPSLoad": 100,
"MinBattVoltage": 6,
"MaxBattVoltage": 18,
"MinimizeToTray": "False",
"MinimizeOnStart": "False",
"CloseToTray": "False",
"StartWithWindows": "False",
"UseLogFile": "False",
"Log Level": 0,
"ShutdownLimitBatteryCharge": 30,
"ShutdownLimitUPSRemainTime": 120,
"ImmediateStopAction": "False",
"Follow_FSD": "False",
"TypeOfStop": 0,
"DelayToShutdown": 15,
"AllowExtendedShutdownDelay": "False",
"ExtendedShutdownDelay": 15,
"VerifyUpdate": "False",
"VerifyUpdateAtStart": "False",
"DelayBetweenEachVerification": 2,
"StableOrDevBranch": 0,
"LastDateVerification": ""
}

Error Message :
Exception type: System.ArgumentOutOfRangeException
Exception message: Value '0' is not a valid value for Interval. Interval must be greater than 0.
Parameter name: Interval
Exception stack trace:
at System.Windows.Forms.Timer.set_Interval(Int32 value)
at WinNUT_Client.WinNUT.WinNUT_Load(Object sender, EventArgs e)
at System.EventHandler.Invoke(Object sender, EventArgs e)
at System.Windows.Forms.Form.OnLoad(EventArgs e)
at System.Windows.Forms.Form.OnCreateControl()
at System.Windows.Forms.Control.CreateControl(Boolean fIgnoreVisible)
at System.Windows.Forms.Control.CreateControl()
at System.Windows.Forms.Control.WmShowWindow(Message& m)
at System.Windows.Forms.Control.WndProc(Message& m)
at System.Windows.Forms.ScrollableControl.WndProc(Message& m)
at System.Windows.Forms.ContainerControl.WndProc(Message& m)
at System.Windows.Forms.Form.WmShowWindow(Message& m)
at System.Windows.Forms.Form.WndProc(Message& m)
at System.Windows.Forms.Control.ControlNativeWindow.OnMessage(Message& m)
at System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)

Last Events :
3/4/2024 8:30:41 AM Pid: 15784 WinNUT : Initialisation Globals Variables Complete
3/4/2024 8:30:41 AM Pid: 15784 WinNUT : Initialisation Params Complete
3/4/2024 8:30:41 AM Pid: 15784 WinNUT : Loaded Params Complete
3/4/2024 8:30:41 AM Pid: 15784 WinNUT : NotifyIcons Initialised
3/4/2024 8:30:41 AM Pid: 15784 WinNUT : Windows 10 Toast Notification Not Available. Too Old Windows Version
--------------------------------
I think I will uninstall it, then try to configure it again, without the login info.

I will post if that is successful. In the meantime, hopefully someone will find this useful.  

Thanks!

**Comment by gbakeman** [comment, marked as answer]
Hi,

Thanks for moving this over here. I believe this issue was fixed a long time ago, please download the latest version and check again.

**Comment by DaGwhite** [comment]
The version I installed was v2.1.7740.35837, which I found from a "latest available Releases" link here:
[https://github.com/gawindx/WinNUT-Client/tree/master?tab=readme-ov-file](url)  
However, I just noticed the release is dated March of 2021...

I just found "Pre-Release v2.3.8824" under [https://github.com/nutdotnet/WinNUT-Client](url).  I'll try that this time...

Thanks!

**Comment by DaGwhite** [comment]
That worked MUCH better!  THANK YOU!

Somebody needs to break the tines off the old fork...

**Reply by gbakeman** [reply]
I'm glad it worked! I've been working to try and separate us from the old repository. Would you mind sharing how you initially found WinNUT?

**Comment by DaGwhite** [comment]
No problem!  I did a Google search on WinNut Client, and the first one that came up for me was the old fork.  I picked the top link, and off I went down the rabbit hole...

**Reply by gbakeman** [reply]
OK, makes sense. We're working out how to spread the word in #91, feel free to contribute there if you have any ideas.
